packages/service/src/utils/passport.ts

```diff
@@ -7,7 +7,7 @@ passport.use(
     try {
       const user = await validateUser({ name: username, password: password });
       if (!user) {
-        return done("invalid user");
```

Comment:
why is this being removed?

Reply:
it's being replaced with done(null,null) so that the application is sent to the failureRedirect